backend/src/main/kotlin/app/ehrenamtskarte/backend/application/database/Setup.kt

```diff
@@ -0,0 +1,9 @@
+package app.ehrenamtskarte.backend.application.database
+
+import org.jetbrains.exposed.sql.SchemaUtils
+
+fun setupDatabase() {
+    SchemaUtils.create(
+        EakApplications
+    )
```

Comment:
Are we maybe creating the other schemata elsewhere? Why is there only one line in this file?

Reply:
This is only the database schema, which only uses a single table. Should be fine afaics